docusaurus/docs/operate/configs/supplier_staking_config.md

```diff
@@ -44,6 +48,42 @@ poktrolld tx supplier stake-supplier \
   --node tcp://poktroll-node:26657
 ```
 
+## Staking types
+
+The `Supplier` staking command supports two types of staking:
+
+### Custodial Staking
+
+The owner of the `Supplier` is the same as the operator.
+This means the account that receives the rewards is the same as the one that
+signs the `RelayResponse`s and submits claims and proofs.
+
+Custodial staking is the simplest to set up and manage, as there is no need to
+manage multiple accounts. It is suitable for `Supplier`s that do not have concerns
+about using the private key of the staking or the rewarded account to operate the
+`RelayMiner` or update the `Supplier`'s stake and services.
+
+### Non-Custodial Staking
+
+The owner of the `Supplier` is different from the operator.
+This means the account that receives the rewards is different from the one signing
+the `RelayResponse`s and submitting claims and proofs.
+
+Non-custodial staking is suitable for `Supplier`s that want to separate the staking
+or the rewarded account (i.e., the account that holds the `upokt` stake or rewards)
+from the account operates the `RelayMiner`.
+
+:::note
+
+When staking a `Supplier`, the signing account specified with the `--from` flag
+(which may differ from the `Supplier`'s owner or operator) will have its `upokt`
+balance deducted to stake the `Supplier`.
+
+When unstaking a `Supplier`, the staked `upokt` will be returned to the `owner_address`
+account.
+
+:::
+
 ## Configuration
 
 ### `owner_address`
```

Comment:
It's not clear to me how "Non-empty" is different from "Required".

Reply:
Since this is neither a map nor a list, there's no difference.

Will remove the `Non-Empty` mention